bundle/manifests/sonataflow.org_sonataflowclusterplatforms.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.15.0
+    controller-gen.kubebuilder.io/version: v0.9.2
```

Comment:
You must update your controller gen locally. Just delete the `bin` directory in the project's root.

Reply:
Done